ext/standard/mail.c

```diff
@@ -74,6 +74,18 @@ static bool php_mail_build_headers_check_field_value(zval *val)
 			}
 			return FAILURE;
 		}
+		/*
+		 * This is a way of writing that violates rfc2822,
+		 * but there are cases where this is accepted.
```

Comment:
I don't think this is correct either - see https://www.rfc-editor.org/rfc/rfc2822#section-2.2 

> A field body may be composed of any US-ASCII characters, except for CR and LF

And it is explicitly prohibited by https://www.rfc-editor.org/rfc/rfc5322#section-2.2

>  field body MUST NOT include CR and LF except when used in "folding" and "unfolding"

Note that folding and unfolding should be only handled by PHP and not the user so it means that is never allowed for the user.